tests/test_data/spell_words.txt

```diff
@@ -167,4 +171,9 @@ Zach
 #------------------------------------
 #------ Stupid stuff to avoid false positive hits
 # Keep as short as possible, use \var{} etc. instead
-
+bd
+dir
+ie
```

Comment:
I guess that this is a spelling error and should be corrected to "i.e." in the documentation?